default.nix

```diff
@@ -155,7 +155,7 @@ in
             "${gems}/bin/bundle exec rails db:migrate"
             "${gems}/bin/bundle exec rails ffmpeg:check_version"
           ];
-          ExecStart = "${gems}/bin/bundle exec puma -C ${api}/config/puma.rb";
+          ExecStart = "${gems}/bin/bundle exec --keep-file-descriptors puma -C ${api}/config/puma.rb";
```

Comment:
Maybe better to just use the puma binstub directly?
```
ExecStart = "${accentor-api.env}/bin/puma -C ${accentor-api}/config/puma.rb"
```

Reply:
Yeah, might be, because this didn't seem to work.